src/Entity/OgRole.php

```diff
@@ -6,177 +6,206 @@
  */
 namespace Drupal\og\Entity;
 
-use Drupal\Core\Entity\ContentEntityBase;
-use Drupal\Core\Entity\EntityTypeInterface;
-use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Config\Entity\ConfigEntityBase;
 
 /**
- * todo: Find a way to attach the roles to group:
- *  - UUID?
- *  - Service?
- *
- * Convert into
- *
- *
- * @ContentEntityType(
+ * @ConfigEntityType(
```

Comment:
Yes. I'm doing the work in small batches. Once this will be merged the scope of #77 will be smaller and much more focused.